core/drivers/corda-driver/docker-compose.yaml

```diff
@@ -13,8 +13,15 @@ services:
       - DRIVER_PORT=${DRIVER_PORT}
       - DRIVER_RPC_USERNAME=${DRIVER_RPC_USERNAME}
       - DRIVER_RPC_PASSWORD=${DRIVER_RPC_PASSWORD}
+      - RELAY_TLS=${RELAY_TLS}
+      - RELAY_TLSCA_CERT_PATHS=${RELAY_TLSCA_CERT_PATHS}
```

Comment:
Shouldn't `RELAY_TLSCA_TRUST_STORE` and `RELAY_TLSCA_TRUST_STORE_PASSWORD` also be added here? to give the user option to chose whichever one they want?

Reply:
Yes, this was an omission. Added now.